Tests/Unit/RealMiniAppTests.swift

```diff
@@ -0,0 +1,117 @@
+import Quick
+import Nimble
+@testable import MiniApp
+
+// swiftlint:disable function_body_length
+class RealMiniAppTests: QuickSpec {
+
+    override func spec() {
+        let miniAppStatus = MiniAppStatus()
+        describe("Real mini app tests") {
+            afterEach {
+                deleteMockMiniApp(appId: mockMiniAppInfo.id, versionId: mockMiniAppInfo.version.versionId)
+                deleteStatusPreferences()
+            }
+            context("when getMiniApp is called with valid app id") {
+                it("will return valid MiniAppInfo") {
+                    let realMiniApp = RealMiniApp()
+                    let mockAPIClient = MockAPIClient()
+                    realMiniApp.miniAppClient = mockAPIClient
```

Comment:
It seems you need to create and set the mocks on each test. Could you do this kind of setup in `beforeEach` instead of in each test?

Reply:
That makes sense, but beforeEach is a closure and the objects declared in it can't be used outside. So I had moved those repeated codes to global now.